test/MorseL.Scaleout.Redis.Tests/RedisBackplaneTests.cs

```diff
@@ -45,6 +45,30 @@ public async void ConnectionSubscriptionAddedOnConnect()
             await actualHub.OnDisconnected(webSocket, null);
         }
 
+        [Fact]
+        public async void ClientConnectAndDisconnectCleansUpBackplaneEventHandlers()
```

Comment:
Once this lands I'll look at doing some `dotMemoryUnit` tests to try to also confirm that the memory is completely cleaned up (since this can only verify that the bookkeeping data structures are correctly bookkept).